ci-droid-gui/src/app/routes/form/form.component.spec.ts

```diff
@@ -417,5 +432,48 @@ describe('FormComponent', () => {
       ]);
       expect(component.resources.length).toEqual(1);
     });
+
+    it('should create the request for bulk update', () => {
+      initializeForm();
+      expect(component.createUpdateRequest().email).toBe('dileep.jami@gmail.com');
```

Comment:
you need to assert for all the values ,no ?

- token
- commitMessage
- the change that has been requested(missing in the initializeForm())

Reply:
Hi Paul,

What is the third one? I did not understand.